### PR TITLE
feat(wallet/frontend): build separate images for prod and staging

### PR DESCRIPTION
**Pull request by raducristianpopa**
### Context

- fixes #

### CODE REVIEW

File: packages/wallet/frontend/src/components/providers/index.tsx [hunk 1]
```diff
@@ -11,19 +11,19 @@ type AppProviderProps = {
...
-    >
-      <MenuProvider>
-        <OnboardingProvider>
-          <PasswordProvider>
-            <DialogProvider>{children}</DialogProvider>
-          </PasswordProvider>
-        </OnboardingProvider>
-      </MenuProvider>
-    </ThemeProvider>
+    // <ThemeProvider
```
Comment (Tymmmy) [inline packages/wallet/frontend/src/components/providers/index.tsx:14]: you want to keep the comments?

Comment (raducristianpopa) [reply packages/wallet/frontend/src/components/providers/index.tsx:14]: That was my initial plan, but now.. No, not going to keep them.

File: .github/workflows/build-publish.yaml [hunk 2]
```diff
@@ -90,6 +107,13 @@ jobs:
...
             ${{ matrix.package.identifier == 'boutique:frontend' && format('VITE_CURRENCY={0}', matrix.package.currency) || '' }}
+            ${{ matrix.package.identifier == 'wallet:frontend' && format('PORT={0}', matrix.package.port) || '' }}
+            ${{ matrix.package.identifier == 'wallet:frontend' && format('COOKIE_NAME={0}', matrix.package.cookie_name) || '' }}
+            ${{ matrix.package.identifier == 'wallet:frontend' && format('NEXT_PUBLIC_BACKEND_URL={0}', matrix.package.backend_url) || '' }}
+            ${{ matrix.package.identifier == 'wallet:frontend' && format('NEXT_PUBLIC_OPEN_PAYMENTS_HOST={0}', matrix.package.open_payments_host) || '' }}
+            ${{ matrix.package.identifier == 'wallet:frontend' && format('NEXT_PUBLIC_AUTH_HOST={0}', matrix.package.auth_host) || '' }}
+            ${{ matrix.package.identifier == 'wallet:frontend' && format('NEXT_PUBLIC_GATEHUB_ENV={0}', matrix.package.gatehub_env) || '' }}
+            ${{ matrix.package.identifier == 'wallet:frontend' && format('NEXT_PUBLIC_THEME={0}', matrix.package.theme) || '' }}
```
Comment (raducristianpopa) [inline .github/workflows/build-publish.yaml:116]: Note to reviewers: please double check these lines.

File: packages/wallet/frontend/next.config.js [hunk 3]
```diff
@@ -2,6 +2,15 @@ const withBundleAnalyzer = require('@next/bundle-analyzer')({
   enabled: process.env.ANALYZE === 'true'
 })
 
+let NEXT_PUBLIC_FEATURES_ENABLED = 'true'
+
+if (
+  process.env.NODE_ENV === 'production' &&
+  process.env.GATEHUB_ENV === 'staging'
```
Comment (dragosp1011) [inline packages/wallet/frontend/next.config.js:9]: staging or sandbox?

Comment (raducristianpopa) [reply packages/wallet/frontend/next.config.js:9]: Oh, it's sandbox in backend. Will update.

**Review by dragosp1011 — APPROVED** [review]

**Pull request merged**